scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/SortedBucketTransform.java

```diff
@@ -216,8 +216,9 @@ public FinalizeTransformedBuckets(
     }
 
     @ProcessElement
-    public void processElement(ProcessContext c) throws IOException {
-      final Iterator<MergedBucket> mergedBuckets = c.element().iterator();
+    public void processElement(@DoFn.Element Iterable<MergedBucket> element,
```

Comment:
🎨 Import `org.apache.beam.sdk.transforms.DoFn.Element` directly and just reference it as `@Element`?